theories/prelude/Logic.ec

```diff
@@ -678,6 +678,11 @@ lemma semptyNP ['a] (E : 'a -> bool) :
   !sempty E <=> exists x, E x.
 proof. by rewrite /sempty -negb_exists. qed.
 
+(* -------------------------------------------------------------------- *)
+lemma pred_0Vmem (p : 'a -> bool) : p = pred0 \/ exists x, p x by smt().
```

Comment:
@strub can you please confirm that this is where this lemma belongs? We're not sure how to split material between the various prelude theories.

Reply:
I would say `Core.ec`

Reply:
@chdoc, can you please move it (next to all the other lemmas about the `predX` operators) and amend/squash?

Reply:
done :heavy_check_mark: